Help e2e tests not fail so often

The e2e tests are failing quite often for [`client-participation/integration.spec.js`](https://github.com/pol-is/polis/blob/dev/e2e/cypress/integration/polis/client-participation/integration.spec.js), with this error:

```
Blocked a frame with origin "https://localhost" from accessing a cross-origin frame
```

Re-running tests seem to resolve, but various units of the test will fail quite often, and succeed other times.

This is the error we're getting, and a recommended simple fix: https://gitlab.com/kgroat/cypress-iframe/-/issues/3#note_331248266

Also, hitting this, and gonna address is same PR, though slightly different:

https://github.com/pol-is/polis/pull/859/checks?check_run_id=1961464575#step:10:404

> AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-test-id="description"]`, but never found it.

Related to cypress-iframe timeout not being long enough: https://gitlab.com/kgroat/cypress-iframe/-/issues/7

--- a/e2e/cypress.json
+++ b/e2e/cypress.json
@@ -1,4 +1,5 @@
 {
+  "chromeWebSecurity": false,
   "apiPath": "/api/v3",
   "ignoreTestFiles": "**/examples/*.spec.js",
   "baseUrl": "http://localhost"

--- a/e2e/package.json
+++ b/e2e/package.json
@@ -15,7 +15,7 @@
   "author": "Benjamin Rosas <ben@aliencyb.org>",
   "devDependencies": {
     "cypress": "^6.2.1",
-    "cypress-iframe": "^1.0.1",
+    "cypress-iframe": "gitlab:patcon/cypress-iframe#7-improve-timeout",
     "cypress-terminal-report": "^1.4.1",
     "eslint": "^7.1.0",
     "eslint-config-prettier": "^6.11.0",

